selftests/checkall

```diff
@@ -7,11 +7,11 @@ run_rc() {
         GR=1
     fi
 }
-run_rc 'inspekt lint'
+run_rc 'inspekt --exclude=.git lint'
```

Comment:
It could be unrelated to this, but...

inspector.py has:

def _read_gitignore(self):
        config_ignore = '{home}/.config/git/ignore'.format(
            home=os.environ.get('HOME'))
        git_dir_ignore = '{gitdir}/.config/git/ignore'.format(
            gitdir=os.environ.get('GIT_DIR'))
        dot_ignore = '.gitignore'
        for ign in (dot_ignore, git_dir_ignore, config_ignore):
            if os.path.isfile(ign):
                with open(ign) as f:
                    self.ignore.extend([x.strip() for x in f.readlines()])
                break
        self.ignore = set(self.ignore)